Sources/PrimerSDK/Classes/Data Models/PrimerInternalSessionFlow.swift

```diff
@@ -128,3 +128,8 @@ internal enum PrimerInternalSessionFlow {
         }
     }
 }
+
+public enum PrimerSessionIntent: String {
```

Comment:
[Feedback]
Feels weird that a `public` enum is in Primer*Internal*SessionFlow.

Reply:
Yep, naming wasn't the best. Renamed the file.